modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -465,7 +471,6 @@ export default {
     // Toolbar handlers
     selectClick() {
       this.selectAll();
-      this.editing = undefined;
```

Comment:
Right to select all, none of them must be selected.

Reply:
same as before, we rely on `updateEditing` in the `checked` watcher